src/editor/editorinteractive.cc

```diff
@@ -410,6 +583,18 @@ void EditorInteractive::rebuild_showhide_menu() {
 	showhidemenu_.add(draw_resources_ ? _("Hide Resources") : _("Show Resources"),
 	                  ShowHideEntry::kResources,
 	                  g_gr->images().get("images/wui/menus/toggle_resources.png"));
+
+	if (finalized_) {
+		/** TRANSLATORS: An entry in the game's show/hide menu to toggle whether building names are
+		 * shown */
+		showhidemenu_.add(get_display_flag(dfShowCensus) ? _("Hide Census") : _("Show Census"),
```

Comment:
In the game, the census entry is directly below the building spaces. We should keep this consistent.

We could also consider putting the ownership layer on top, so that the mouse positioning will be stable when clicking on entries.